powerauth-webflow-client/src/main/java/io/getlime/security/powerauth/app/webflow/demo/controller/HomeController.java

```diff
@@ -61,7 +58,7 @@ public HomeController(Provider<ConnectionRepository> connectionRepositoryProvide
     }
 
     @RequestMapping("/")
-    public String home(Principal currentUser, Model model, HttpSession session) {
+    public String home(Model model, HttpSession session) {
```

Comment:
We should rather stick to one approach. After I've learnt a lesson I am voting for having autowired `HttpSession` also in this controller :).

Reply:
Ok, I will change it.